Notifier-service/src/services/conference.ts

```diff
@@ -3,30 +3,32 @@ import { ConferenceModel } from '../interfaces/models/conference'
 import { Conference } from '../schemas/conferences';
 import { Logger } from '../utility/log';
 import { injectable } from 'inversify';
+import { ConferenceModelMongo } from '../models/conference';
+
 
 @injectable()
 export class ConferenceServiceI extends ConferenceService {
     private logger = new Logger(this.constructor.name).getLogger()
-    constructor(private conferenceModel: ConferenceModel) {
+    constructor(private conferenceModel: ConferenceModel,
+                private conferenceModelMongo:ConferenceModelMongo) {
         super()
     }
-    async getOne(): Promise<Conference | null> {
+    async getOne(_id: string): Promise<Conference | null> {
         this.logger.debug("getOne invoked")
         return new Promise<Conference | null>((resolve, reject) => {
-            this.conferenceModel.getOne().then(value => {
-                if (value == null) {
+            this.conferenceModelMongo.getOne(_id).then((value)=>{
+                if(value == null){
                     resolve(null)
-                }
-                else {
-                    let result = value.toObject()
-                    let conference: Conference = result
+                }else{
+                    let conference: Conference = value.toObject()
```

Comment:
* Check whether this conference object has the key _id in it. If it isn't present ,include it.

Reply:
I have added _id in conference object now